build_android/src/main/res/values-id/strings.xml

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+	<!-- Application Navigation -->
+
+	<string name="about_play">Tentang Play!</string>
+	<string name="about_play_emu">Play! Emulator PlayStation 2</string>
```

Comment:
here should be `Play! PlayStation 2 Emulator`

Reply:
got it